### ISSUE TITLE
The method getX() is undefined for the type Vector3f

### ISSUE THREAD

**Issue by mikeubell** [TheGreyGhost/MinecraftByExample#7]
Loaded up MinecraftByExample with 8.1.  Running in eclipse I get the above compliation error in HUDtextRenderer.java and MenuItemCameraTransforms.java

**Comment by TheGreyGhost** [comment]
That's odd.  Have a look at the top of the HUDtextRenderer.java; does it import a different Vector3f?  
Mine has import javax.vecmath.Vector3f; which is the minecraft version
Would you please check to see which Vector3f your Eclipse is importing & post it back here?

-TGG

**Comment by mikeubell** [comment]
This is strange, I had removed the offending files and commented out the references to them.  I then did a git co -- . and reran 'gradlew eclipse'.  Now I am seeing:
6 instances of:
minecraftbyexample.mbe14_item_camera_transforms cannot be resolved	ClientOnlyProxy.java	/MinecraftByExample/src/main/java/minecraftbyexample	line 34	Java Problem

I don't see an mbe14_item_camera_transforms directory.

I'll try starting over from a fresh clone and see what happens.
